corehq/apps/reports_core/filters.py

```diff
@@ -115,8 +115,8 @@ def date_or_nothing(param):
             return DateSpan(startdate, enddate, inclusive=date_range_inclusive)
 
     def default_value(self):
-        # default to a week's worth of data.
-        return DateSpan.since(7)
+        # default to "Show All Dates"
+        return None
```

Comment:
This feels like a potentially broad-reaching change.  Do you have a sense of what else this will affect?